src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2916,11 +2918,11 @@ public Mode3D get3DLayout() {
 			case "oulf":
 			case "top-bottom (left eye first)":
 				isAnaglyph = false;
-				return Mode3D.OUL;
+				return Mode3D.ABL;
 			case "ourf":
```

Comment:
@Sami32 no because this is the value which is given by MediaInfo. The same apply for `oulf`.